src/panels/lovelace/cards/hui-picture-card.ts

```diff
@@ -79,18 +80,20 @@ export class HuiPictureCard extends LitElement implements LovelaceCard {
       return html``;
     }
 
+    const clickable = Boolean(
+      this._config.tap_action || this._config.hold_action
+    );
+
     return html`
       <ha-card
         @action=${this._handleAction}
         .actionHandler=${actionHandler({
           hasHold: hasAction(this._config!.hold_action),
           hasDoubleClick: hasAction(this._config!.double_tap_action),
         })}
-        tabindex="0"
+        tabindex=${ifDefined(clickable ? 0 : undefined)}
```

Comment:
hitting enter currently only supports tap_action, so should be the only one considered for setting tabindex.

Reply:
`hasAction(this._config.tap_action)`